ts/IterableInboxMessageList.tsx

```diff
@@ -1,31 +1,25 @@
-'use strict';
-import React, { ReactElement } from 'react';
-import { ScrollView, Text, StyleSheet } from 'react-native';
-import IterableInboxMessageCell from './IterableInboxMessageCell';
+'use strict'
+import React, { ReactElement } from 'react'
+import { ScrollView, Text, StyleSheet } from 'react-native'
+import IterableInboxClickableRow from './IterableInboxClickableRow'
 
 type MessageListProps = {
    messages: Array<any>,
    handleMessageSelect: Function  
 }
 
 const IterableInboxMessageList = ({ messages, handleMessageSelect }: MessageListProps) => {
-   function flaggedMessage(message: { [key: string]: any }, index: number) {
-      return (index === messages.length - 1) ?
-         <IterableInboxMessageCell
-            index={index} 
-            message={message}
-            handleMessageSelect={(index: number) => handleMessageSelect(index, messages)} 
-            last={true} /> :
-         <IterableInboxMessageCell
-            index={index} 
-            message={message}
-            handleMessageSelect={(index: number) => handleMessageSelect(index, messages)}  
-            last={false} />
-   }
 
    function displayMessages() {
       return messages.map((message, index) => {
-         return flaggedMessage(message, index)
+         let last = (index === messages.length - 1) ? true : false
+         return (         
+            <IterableInboxClickableRow
+               key={index}
```

Comment:
does a message have an `id` property, or any other unique identifier? if possible, you'll want to rely on something more stable than the index of an array to act as your `key` -- this should be your last resort if there are no better identifiers.

Reply:
I will use the message ID for the key,